### PR TITLE
Support `right` position

### PR DESCRIPTION
**Pull request by jsw0528**
Percentage of `left` is not precision.

### CODE REVIEW

File: spin.js [hunk 1]
```diff
@@ -165,6 +168,7 @@
   , className: 'spinner'  // CSS class to assign to the element
   , top: '50%'            // center vertically
   , left: '50%'           // center horizontally
+  , right: null           // center horizontally
```
Comment (mikesherov) [inline spin.js]: Incorrect comment.

File: spin.js [hunk 2]
```diff
@@ -22,6 +22,7 @@
     , className: 'spinner'  // CSS class to assign to the element
     , top: '50%'            // center vertically
     , left: '50%'           // center horizontally
+    , right: null           // center horizontally
```
Comment (mikesherov) [inline spin.js]: Incorrect comment.

File: spin.js (hunk 1 above)
Comment (jsw0528) [inline spin.js]: Fixed.

File: spin.js (hunk 2 above)
Comment (jsw0528) [inline spin.js]: Fixed.

**Comment by tirsen** [comment]
+1

**Comment by theodorejb** [comment]
Duplicate of #330.

**Comment by jsw0528** [comment]
330 > 299

hehe

**Pull request closed without merging**